velox/core/QueryConfig.h

```diff
@@ -192,6 +192,18 @@ class QueryConfig {
   static constexpr const char* kSparkLegacySizeOfNull =
       "spark.legacy_size_of_null";
 
+  // The default number of expected items for the runtime bloomfilter.
+  static constexpr const char* kSparkRuntimeBloomFilterExpectedNumItems =
```

Comment:
It would be nice to split fragment changes (add access to query config for aggregate functions) from using that functionality into separate PRs.

Reply:
I put here because I can add unit test to make sure the functionality works well with these configs. Do I need to move it to another PR and no test for the functionality?

Reply:
This is a large PR. Having multiple logical changes combined here makes it hard to review and will be difficult to land. Let's limit this PR to just passing config to scalar, aggregate and window functions.